server/channels/app/helper_test.go

```diff
@@ -278,7 +276,8 @@ func (th *TestHelper) InitBasic() *TestHelper {
 	th.BasicUser2 = userCache.BasicUser2.DeepCopy()
 
 	users := []*model.User{th.SystemAdminUser, th.BasicUser, th.BasicUser2}
-	mainHelper.GetSQLStore().User().InsertUsers(users)
+	err := mainHelper.GetSQLStore().User().InsertUsers(users)
+	require.NoError(th.TB, err)
```

Comment:
We can remove the caching via `sync.Once`. It is not helpful and only complicates the code further.